javascript/funWithFunctions.md

```diff
@@ -0,0 +1,24 @@
+## Functional Expressions in D&D ##
+I've been having fun with functions in creating a JavaScript version of the 5th edition D&D character sheet.
+
+In recreating the character bonus table I got thru 47 lines of inline conditionals before the formula slapped me in the face:  
+![](http://cl.ly/image/3046350l3X1r/OG_s.jpg)
+
+
+Even in adding abilities to an array, I was still left with 6 variables and a declared function:
+
+	var abilities = [9, 10, 15, 12, 9, 19];  //The [str, dex, con, int, wis, chr] of my current character
```

Comment:
You can get sick highlighting by doing this:

\`\`\`javascript
var abilities = ...your code
\`\`\`

i.e. instead of indenting by 4 spaces, don't indent at all and wrap it in the special triple-backticks syntax.